CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Menu] Don't subscribe to unhealthy VDI chains if user has not enough permissions (PR [#7265](https://github.com/vatesfr/xen-orchestra/pull/7265))
```

Comment:
What visible change does this PR bring to the users?

Reply:
@julien-f It removes error messages from logs

Reply:
Then this is what you should highlight in the CHANGELOG.

Commit message is for dev, CHANGELOG is for users.